Source/Core/VideoBackends/Metal/MTLUtil.h

```diff
@@ -0,0 +1,56 @@
+// Copyright 2022 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <Metal/Metal.h>
+#include <vector>
+
+#include "VideoCommon/AbstractShader.h"
+#include "VideoCommon/TextureConfig.h"
+#include "VideoCommon/VideoConfig.h"
+
+#include "VideoBackends/Metal/MRCHelpers.h"
+
+namespace Metal
+{
+struct DeviceFeatures
+{
+  // clang-format off
+  bool unified_memory : 1;
```

Comment:
`BitField`?

Reply:
I'm happy to use non-bitfield, whole-byte bools here if you prefer